maintainers/README.md

```diff
@@ -58,10 +58,33 @@ person as a reviewer, and then waiting a week for a reaction.
 
 The maintainer is welcome to come back at any time.
 
-### Tools for maintainers
+## Tools for maintainers
 
-When a pull request is made against a package, OfBorg will notify the
-appropriate maintainer(s).
+When a pull request is made against a package, nixpkgs CI will notify the
+appropriate maintainer(s) based on the package names listed in the commit titles.
+
+Maintainers can also invoke the [nixpkgs-merge-bot](https://github.com/nixos/nixpkgs-merge-bot)
+to merge pull requests targeting packages they are the maintainer of which satisfy the current
+security [constraints](https://github.com/NixOS/nixpkgs-merge-bot/blob/main/README.md#constraints).
+
+New maintainers will automatically get invited to join the [NixOS/nixpkgs-maintainers](https://github.com/orgs/NixOS/teams/nixpkgs-maintainers) GitHub team.
+By joining it you will get some triaging rights in the nixpkgs repository, like the ability to add and remove labels.
+However, the GitHub team invite is usually only sent by email and is only valid for one week!
+Should it expire please ask for a re-invite in the [GitHub org owners help desk Matrix channel](https://matrix.to/#/#org_owners:nixos.org).
+
+[zh.fail](https://zh.fail/failed/overview.html) tracks all package build failures on `master` grouped by maintainer.
+
+#### Unofficial maintainer tooling
+
+[asymmetric/nixpkgs-update-notifier](https://github.com/asymmetric/nixpkgs-update-notifier) is a matrix
+bot that scrapes the [nixpkgs-update logs](https://nixpkgs-update-logs.nix-community.org/) and notifies
+you if nixpkgs-update/@r-ryantm fails to update any of the packages you've subscribed to.
+
+[repology.org](https://repology.org) tracks and compares the versions of packages between various package repositories, letting you know what packages may be out of date or insecure.
+You can view what packages a specific maintainer maintains and subscribe to updates with atom/rss. Example: [repology.org/maintainer/pbsds](https://repology.org/maintainer/pbsds%40hotmail.com).
+
+[nixpk.gs/pr-tracker](https://nixpk.gs/pr-tracker.html) and [nixpkgs-tracker.ocfox.me](https://nixpkgs-tracker.ocfox.me/)
```

Comment:
I would then also mention this tool: https://nixpkgs-tracker.ocfox.me/ (I personally prefer it)

Reply:
already mentioned. i made sure to include it since it is not limited by the api restrictions of the gh token on the server

Reply:
Oops, sorry !

Reply:
Neither is pr-tracker in practice. The rate limit is 5000 requests per hour, which I would be shocked if it's ever been hit. If something goes wrong with pr-tracker it tends to be git garbage collection having failed and disk space having run out as a result.

Reply:
No problem. 🥰